src/app/cube/user-profile/components/edit-profile/edit-profile.component.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  Component,
+  OnChanges,
+  EventEmitter,
+  Output,
+  Input
+} from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+import { AuthService } from 'app/core/auth.service';
+import { ProfileService } from 'app/core/profiles.service';
+import { ToastrOvenService } from 'app/shared/modules/toaster/notification.service';
+import { environment } from '@env/environment';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'clark-edit-profile',
+  templateUrl: './edit-profile.component.html',
+  styleUrls: ['./edit-profile.component.scss']
+})
+export class EditProfileComponent implements OnChanges {
+  ssoRedirect = environment.apiURL + '/google';
+  elementRef: any;
+  @Input() user;
+  @Input() gravatarImage;
+  @Output() close: EventEmitter<void> = new EventEmitter();
+  @Output() userInfo: EventEmitter<{}> = new EventEmitter();
+  errorMsg = 'There was an issue on our end with your registration, we are sorry for the inconvience.\n Please try again later!';
```

Comment:
Should this error message be changed to reflect the user updating their profile and not registering?

Reply:
I actually don't use this so I'll delete it